tools/code-analyzer/README.md

```diff
@@ -0,0 +1,32 @@
+# Code Analyzer
```

Comment:
A brief README that explains commands and shows users how to find more with `--help`